### PR TITLE
SeaweedFS Remote Mount

### PR DESCRIPTION
**Pull request by yashgorana**
## Description

Closes https://github.com/OpenMined/Heartbeat/issues/1149

## Affected Dependencies
List any dependencies that are required for this change.

## How has this been tested?
- Describe the tests that you ran to verify your changes.
- Provide instructions so we can reproduce.
- List any relevant details for your test configuration.

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [x] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [x] My changes are covered by tests

### CODE REVIEW

File: packages/grid/helm/syft/templates/backend/backend-statefulset.yaml [hunk 1]
```diff
@@ -100,8 +104,6 @@ spec:
                 secretKeyRef:
                     name: {{ .Values.seaweedfs.secretKeyName | required "seaweedfs.secretKeyName is required" }}
                     key: s3RootPassword
-            - name: S3_PORT
```
Comment (rasswanth-s) [inline packages/grid/helm/syft/templates/backend/backend-statefulset.yaml:103]: Q: Why did we remove the S3_PORT ?

Comment (rasswanth-s) [reply packages/grid/helm/syft/templates/backend/backend-statefulset.yaml:103]: Probably, as there is no extra usefulness, is allowing users to customize this port?

File: tox.ini [hunk 2]
```diff
@@ -1059,3 +1060,16 @@ commands =
     fi"
 
     pytest notebooks/api/0.8 --nbmake -p no:randomly -vvvv --nbmake-timeout=1000 -k '{env:EXCLUDE_NOTEBOOKS:}'
+
+[testenv:seaweedfs.test.unit]
+description = Syft Unit Tests
```
Comment (rasswanth-s) [inline tox.ini]: NITPICK: We could rename: Syft Unit Tests ==> Seaweedfs Unit tests

File: packages/grid/helm/syft/templates/backend/backend-statefulset.yaml (hunk 1 above)
Comment (yashgorana) [reply packages/grid/helm/syft/templates/backend/backend-statefulset.yaml:103]: Not needed & not used. No point customizing the port as we're in a container environment and they can just port forward whatever to 8333

File: tox.ini (hunk 2 above)
Comment (yashgorana) [reply tox.ini]: +1

File: packages/grid/helm/syft/templates/backend/backend-statefulset.yaml (hunk 1 above)
Comment (yashgorana) [reply packages/grid/helm/syft/templates/backend/backend-statefulset.yaml:103]: You can still provide it as through `backend.env` in values.yaml

**Review by rasswanth-s — APPROVED** [review]

File: packages/grid/helm/syft/values.yaml [hunk 3]
```diff
@@ -37,34 +37,57 @@ frontend:
   resources: null
 
 seaweedfs:
-  # S3 settings
+  # Master
+  master:
+    # Seaweed Volume Size in MB set on master. This is not PVC.
+    volumeSizeMB: 1000
```
Comment (rasswanth-s) [inline packages/grid/helm/syft/values.yaml:43]: Q: 1. This the maximum capacity of all the Volume servers overall?
      2. Can we control the number of volume servers spinned up?

**Pull request merged**